### DISCUSSION TITLE
My flask project not working on vercel

### DISCUSSION THREAD

**Discussion by yk70840** [vercel/community#6518] [Help]
### Summary

I am deploying my flask app on vercel but its not getting deployed. It is a ml app which predict diseases in plants.
Link to github repo - https://github.com/yk70840/plant_disease 

### Steps to Reproduce

![Screenshot 2024-04-15 203914](https://github.com/vercel/community/assets/36003284/5e3bb0ef-2690-4fb1-a742-ac08fe14312a)

**Comment by versecafe** [comment]
@yk70840 I saw you're using pytorch do you know how large the actual dependencies are, you might be crossing the size limit for lambda functions, from the docs I've been able to find and past use of pytorch you may need to go directly with AWS and use ECR to have a docker container with all of the pytorch dependencies ready to go, you may also want to look at running on a light weight runtime running a compiled and quantized version of the model.

**Reply by yk70840** [reply]
Bro I added a wsgi file and changed the route in vercel.json to wsgi.py file and it's deployed but instead of showing me the page when I open the app link it downloads the wsgi.py file 

![IMG20240416022938.jpg](https://github.com/vercel/community/assets/36003284/f437cb23-b202-44be-8b4f-92b9f66a7c5a)
